lib/matplotlib/backends/backend_qt5.py

```diff
@@ -714,8 +712,7 @@ def _init_toolbar(self):
         # will resize this label instead of the buttons.
         if self.coordinates:
             self.locLabel = QtWidgets.QLabel("", self)
-            self.locLabel.setAlignment(
-                    QtCore.Qt.AlignRight | QtCore.Qt.AlignTop)
+            self.locLabel.setAlignment(QtCore.Qt.AlignRight)
```

Comment:
This doesn't change the alignment for me. I have to explicitly set
```suggestion
            self.locLabel.setAlignment(QtCore.Qt.AlignRight | QtCore.Qt.AlignCenter)
```

Reply:
I think this should be AlignVCenter, sure, that's fine.